virttest/utils_libvirt/libvirt_misc.py

```diff
@@ -0,0 +1,28 @@
+"""
+Virsh command related utility functions which do not belong to special
```

Comment:
I wonder if we can remove the limitation of 'virsh command' here? If we decide to create a libvirt_misc module, I think it can be used  more widely. How do you think?

Reply:
Currently the parsing logic can only meet certain output formats. I am ok to remove the words 'virsh command' here, but for the function created below, I need to add docstring to say currently this function might only support certain output formats, for example, some virsh command results. What do you think?